.gitignore

```
@@ -1,6 +1,6 @@
 /.idea
 /.gradle
-build/
+**/build
```

```suggestion
**/build/
```
завершающий слэшь важен. Иначе у вас будут игонириться папки типа /buildString